src/language/lang_es.php

```diff
@@ -66,7 +66,7 @@
   'Disclaimer' =>
     'Disclaimer',
   'By participating in the contest, you agree to release Facebook and its employees, and the hosting organization from any and all liability, claims or actions of any kind whatsoever for injuries, damages or losses to persons and property which may be sustained in connection with the contest. You acknowledge and agree that Facebook et al is not responsible for technical, hardware or software failures, or other errors or problems which may occur in connection with the contest.' =>
-    'Al participar en el concurso, usted acepta liberar a Facebook y sus empleados, y a la organizadora del evento de cualquier responsabilidad, reclamos o acciones de cualquier tipo por lesiones, daños o pérdidas a las personas y bienes que puedan ir en relación con el concurso. Usted reconoce y acepta que Facebook y otros no se hacen responsables de fallos técnicos, de hardware o software, u otros errores o problemas que pueden ocurrir relacionados al concurso',
+    'Al participar en la competencia, usted acepta liberar a Facebook y sus empleados, y a la organizadora del evento de cualquier responsabilidad, reclamo(s) o accion(es) de cualquier tipo por lesiones, daños o pérdidas a las personas y bienes que puedan ir en relación con el concurso. Usted reconoce y acepta que Facebook y otros no se hacen responsables de fallos técnicos, de hardware o software, u otros errores o problemas que pueden ocurrir relacionados al concurso',
```

Comment:
Again, I prefer to use "competición".
Regarding "reclamos" and "acciones", I don't see the reason to change them.